lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1415,6 +1416,19 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :description => 'Configures the TCP/IP port for the trace observer Host'
         },
         # Instrumentation
+        :'instrumentation.active_support_notifications.active_support_events' => {
```

Comment:
I like how this is scoped for the specific GTSE. 

I wonder if we want to organize this config to support all active support events in one allowlist/denylist kind of like we have set up for our include/exclude lists with other configs. I'm not sure if a big list or one scoped to the specific instrumentation is more user-friendly.

Reply:
My initial thought here was to have the `active_support_events` part of the `:'instrumentation.active_support_notifications.active_support_events'` parameter be different for every different notifications based instrumentation option.

But I can see benefits to holistically re-evaluating our entire design. I can see some good pros that could come out of big lists and fewer subscriptions if we could define a good generic central class whose instance would be associated with the subscriptions.

So I think I'd like to either keep a list per instrumentation that we have a dedicated separate `disable_` parameter for, or start fresh looking at notifications without any baggage from the existing codebase and see what new consolidated patterns we could come up with for both speed and ease of use.

Reply:
Nice! If we see some performance or other user-facing benefits with a consolidated pattern, perhaps we could pitch that as a cornerstone feature of a 10.x major release.

I agree with what you're saying and want to move forward with your current design.

Reply:
Created https://github.com/newrelic/newrelic-ruby-agent/issues/2616